pkg/jsonmessage/jsonmessage_test.go

```diff
@@ -23,16 +30,181 @@ func TestProgress(t *testing.T) {
 		t.Fatalf("Expected %q, got %q", expected, jp2.String())
 	}
 
+	expectedStart := "[==========>                                        ]     20 B/100 B"
+	jp3 := JSONProgress{Current: 20, Total: 100, Start: time.Now().Unix()}
+	// Just look at the start of the string
+	// (the remaining time is really hard to test -_-)
+	if jp3.String()[:len(expectedStart)] != expectedStart {
```

Comment:
Well no, I've set up ``Start`` that is not setup for the 50%. I'm testing the start because I found it really hard to compute the rest of the message (the remaining time). I could give it another try though :P